### DISCUSSION TITLE
How to connect Memospot client to custom Memos server?

### DISCUSSION THREAD

**Discussion by ghost** [memospot/memospot#105] [General]
Hi! I like the idea of a memos app, but I cannot connect to my self-hosted Memos server. How can I do that?

**Comment by lincolnthalles** [comment]
Sorry for the late reply. I've just added that feature on [v0.1.6](https://github.com/memospot/memospot/releases/latest). It's a matter of enabling it on `memospot.yaml`.

Enabling this will prevent the bundled Memos server from starting and Memospot will just act as a web view for the provided server. Note that it will still check if the Memos server is ready before redirecting (via the `/healthz` API endpoint), so it's not possible to load arbitrary web apps.

> [!TIP]
> `memospot.yaml` [About the configuration file](https://memospot.github.io/configuration.html#configuration-location)
> ```yaml
> remote:
>   enabled: true
>   url: https://demo.usememos.com/
>```
>
